src/ast/stmt/EnumStmt.php

```diff
@@ -1,91 +0,0 @@
-<?php
```

Comment:
enum dies?

Reply:
Yes, it is replaced by `type`, which, which is a lot better.